modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc

```diff
@@ -255,16 +264,18 @@ The Mule runtime engine translates the LATEST dependency keyword to the current
 To obtain the most up-to-date `pom.xml` file information, access 
 the connector in https://www.mulesoft.com/exchange/[Anypoint Exchange] and click *Dependency Snippets*.
 
-== Example: Get, Delete, Query, and Execute
+== Example: Create, Get, Query, Update, and Execute
```

Comment:
If these are peers in this heading, is there any reason to separate "execute" as a "function" vs. the other four as "entities" in the opening text?

Reply:
Well, in SAP S/4HANA there are many OData APIs (services) which consist of _entities_ and _service operations_. The create/get/update/delete (CRUD) entities refers to operations you can perform on _entities_, while the execute function refer to _service operation_ which is exposed by an OData service.

Reply:
The style doesn't allow for slashes in text (although fine in file names) - I put these back as comma separated.